client/src/utils/services.ts

```diff
@@ -24,4 +23,20 @@ export const fetchFromApi = async (url: string, body: string) => {
     }
 
     return data;
-}
\ No newline at end of file
+}
+
+export const getFromApi = async (url: string) => {
+
+    const response = await fetch(url);
+    if (!response.ok) {
+        // let message = "get from API failed, error occurred";
+        // if (response.error) {
+        // //     message = data.message;
+        // // } else {
+        // //     message = data;
+        // // }
+        return { error: true, response };
+    }
+    const data = await response.json();
+    return data;
+};
```

Comment:
Add extra line at the end of file.
```suggestion
};

```